fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -14,9 +14,12 @@ def self.start
 
       FastlaneCore::UpdateChecker.start_looking_for_update('fastlane')
       Fastlane.load_actions
-      Fastlane.plugin_manager.load_plugins
-      # *after* loading the plugins
-      Fastlane::PluginUpdateManager.start_looking_for_updates
+      # do not use "include" as it may be some where in the commandline where "env" is required, therefore explicit index->0
+      unless ARGV[0] == "env"
+        # *after* loading the plugins
+        Fastlane.plugin_manager.load_plugins
```

Comment:
actually this one prints the loaded plugins table - and we cannot set a env on `:env` command,
as `self.start` is run before commander! - if we don't suppress this, then the `Terminal-Table`  
of the loaded plugins will be included when using `fastlane env|pbcopy` - wich would mess up issues.
the plugins later on will be loaded in the `environment_printer` - with a `ENV` to suppress table :smile: